semantic_steve/cli.py

```
@@ -0,0 +1,281 @@
+def cli_loop(bot, functionRegistry):
+    
+    run_cli = True
+    
+    @On(bot, "end")
+    def on_end(*args):
+        nonlocal run_cli
+        print("ended")
+        run_cli = False
+        
+    @On(bot, "error")
+    def on_end(*args):
```

Ruff (0.8.2)

Redefinition of unused `on_end` from line 159

(F811)